src/open-hackathon-server/Kaiyuanshe.OpenHackathon.Server/Controllers/UserController.cs

```diff
@@ -65,5 +67,24 @@ public class UserController : HackathonControllerBase
             return Ok(userInfo);
         }
         #endregion
+
+        #region GetUploadToken
+        /// <summary>
+        /// Get file upload token
+        /// </summary>
+        /// <returns>The team</returns>
+        /// <response code="200">Success. The response return a token.</response>
+        [HttpPut]
+        [ProducesResponseType(typeof(Team), StatusCodes.Status200OK)]
+        [SwaggerErrorResponse(400, 404, 412)]
+        [Route("user/storageToken")]

```

Comment:
Let' use `Route("file/generateSasToken")`. It's not for `user`, just a general file uploading api

Reply:
yeah. we can do that. 

I placed it under the user scope, because the file upload route in the previous version was `/api/user/file`: https://github.com/kaiyuanshe/open-hackathon/blob/e03a15762381e96ba679c0d3fe37b5cfef150529/open-hackathon-server/src/hackathon/views/__init__.py#L53

I thought this may require less change in frontend.